app/domain/logs.rb

```diff
@@ -277,65 +277,65 @@ module AuthnJwt
 
       IssuerResourceNameConfiguration = ::Util::TrackableLogMessageClass.new(
         msg: "\"issuer\" value will be taken from '{0-resource-id}'",
-        code: "CONJ00054D"
+        code: "CONJ00054I"
       )
 
       RetrievedIssuerValue = ::Util::TrackableLogMessageClass.new(
         msg: "Retrieved \"issuer\" with value '{0}'",
-        code: "CONJ00055D"
+        code: "CONJ00055I"
       )
 
       ParsingIssuerFromUri = ::Util::TrackableLogMessageClass.new(
         msg: "Parsing \"issuer\" value from '{0}'...",
         code: "CONJ00056D"
       )
 
-      JWTAuthenticatorEntryPoint = ::Util::TrackableLogMessageClass.new(
-        msg: "Started JWT authentication flow for vendor '{0}'",
-        code: "CONJ00057D"
+      JwtAuthenticatorEntryPoint = ::Util::TrackableLogMessageClass.new(
+        msg: "Started JWT authentication flow for authenticator '{0-authenticator-name}'",
+        code: "CONJ00057I"
       )
 
-      URL_IDENTITY_PROVIDER_SELECTED = ::Util::TrackableLogMessageClass.new(
-        msg: "The JWT ID is included in and will be retrieved from the URL",
+      SelectingIdentityProviderInterface = ::Util::TrackableLogMessageClass.new(
+        msg: "Selecting identity provider interface...",
         code: "CONJ00058D"
       )
 
-      DECODED_TOKEN_IDENTITY_PROVIDER_SELECTED = ::Util::TrackableLogMessageClass.new(
-        msg: "JWT Identity in decoded token is available and will be retrieved from it",
-        code: "CONJ00059D"
+      SelectedIdentityProviderInterface = ::Util::TrackableLogMessageClass.new(
+        msg: "Selected identity provider interface: '{0-identity-provider-interface-name}'",
+        code: "CONJ00059I"
       )
 
-      LOOKING_FOR_IDENTITY_FIELD_NAME = ::Util::TrackableLogMessageClass.new(
+      LookingForIdentityFieldName = ::Util::TrackableLogMessageClass.new(
         msg: "Looking for variable field name in '{0}'...",
         code: "CONJ00060D"
       )
 
-      CHECKING_IDENTITY_FIELD_EXISTS = ::Util::TrackableLogMessageClass.new(
+      CheckingIdentityFieldExists = ::Util::TrackableLogMessageClass.new(
         msg: "Checking if field '{0}' in the token...",
         code: "CONJ00061D"
       )
 
-      CREATING_AUTHENTICATION_PARAMETERS_OBJECT = ::Util::TrackableLogMessageClass.new(
+      CreatingAuthenticationParametersObject = ::Util::TrackableLogMessageClass.new(
         msg: "Creating authentication parameters objects...",
         code: "CONJ00062D"
       )
 
-      VALIDATE_AND_DECODE_TOKEN = ::Util::TrackableLogMessageClass.new(
-        msg: "Validate and decode token",
+      CallingValidateAndDecodeToken = ::Util::TrackableLogMessageClass.new(
+        msg: "Calling validate and decode token",
```

Comment:
???

Reply:
what are we doing here?

Reply:
We have 3 main flows in the authentication process:
1. validate and decode
2. get JWT identity
3. Validate restrictions 

I want to be able to identify the start and end of each flow above,
Since we have a lot of "validate and decode messages" in the logs, for example:
```
	CONJ00063D Validate and decode token - main flow authenticator.rb
	CONJ00081D Validating token - From validate and decode command class
	CONJ00083D Validating token signature - From validate and decode command class
	CONJ00005D Token decoded successfully - from verify_and_decode_token.rb
	CONJ00084D Successfully validated token signature -  - From validate and decode command class
	CONJ00085D Validating token claims… - From validate and decode command class
	CONJ00005D Token decoded successfully - from verify_and_decode_token.rb
	CONJ00086D Successfully validated token claims - From validate and decode command class
        CONJ00082D Successfully validated token - From validate and decode command class
```

I wanted to create an convention of start and end of main flows, and i choose to use:
 Calling <flow name>
 Successfully passed  <flow name>
 
 WDYT?

Reply:
Let's discuss this one....each one...